multipatch_analysis/pipeline/morphology.py

```diff
@@ -44,7 +70,32 @@ def create_db_entries(cls, job_id, session):
 
             results = {
                 'pyramidal': pyramidal,
+                'cortical_layer': cortical_layer,
+                'cell_specimen_id': cell_specimen_id,
+                'morpho_db_hash': hash(None),
+
             }
+            
+            if cell_morpho is not None:
+                morpho_db_hash = hash(';'.join(filter(None, cell_morpho)))
+                results['morpho_db_hash'] = morpho_db_hash
+                for morpho_db_name, result in col_names.items():
+                    col_name = result['name']
+                    col_type = result['type']
+                    data = cell_morpho.__getattribute__(morpho_db_name)
```

Comment:
Preferred: `getattr(cell_morpho, morpho_db_name)`

Reply:
for some reason I feel like I couldn't get that to work, but let me try again